ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitterSpec.scala

```diff
@@ -254,6 +263,123 @@ class TransactionCommitterSpec extends WordSpec with Matchers with MockitoSugar
     }
   }
 
+  "rejectionReasonForValidationError" should {
```

Comment:
To improve readability, can you split this test case into separate inconsistent vs. disputed cases, please? You might also want to move the helper defs below the test cases for the previous reasons.

Reply:
I've split the test case into several subtests. IMO there's little point in moving the helper defs into the global test scope because they are really specific to this particular test case.

Reply:
Thank you!